src/SourceGenerators/Uno.UI.SourceGenerators/XamlGenerator/XamlFileGenerator.cs

```diff
@@ -3116,14 +3132,19 @@ private void BuildExtendedProperties(IIndentedStringBuilder outerwriter, XamlObj
 
 			var extendedProperties = GetExtendedProperties(objectDefinition);
 			bool hasChildrenWithPhase = HasChildrenWithPhase(objectDefinition);
-			var isFrameworkElement = IsFrameworkElement(objectDefinition.Type);
-			var hasIsParsing = HasIsParsing(objectDefinition.Type);
+			var objectDefinitionType = FindType(objectDefinition.Type);
+			var isFrameworkElement = IsType(objectDefinitionType, _frameworkElementSymbol);
```

Comment:
Let's make `IsFrameworkElement` use _frameworkElementSymbol is not already the case.

Reply:
@jeromelaban I updated `IsFrameworkElement` in https://github.com/unoplatform/uno/commit/9f2349ed37090b51096abb1fae22eca76f8228b3. I think you mean to revert `IsType` back to `IsFrameworkElement` right? I'll address that soon.

Reply:
@jeromelaban Reverting back to `IsFrameworkElement` will require introducing a new overload to `IsFrameworkElement`.

The changes in this line are not only using `_frameworkElementSymbol`, but also using an ISymbol (`objectDefinitionType`) instead of a XAML type (`objectDefinition.Type`)